docs/sidebars.yml

```diff
@@ -0,0 +1,146 @@
+---
```

Comment:
It's not related to the docs. It's barely parseable by Gnoweb.

Please, consider my proposal to use numbered prefixes files and completely remove this file.

If you have valid reasons for not using my suggestion, please create a YAML file for documentation in the "docs/" directory. The file should contain only essential information relevant to docs/, editors, and gnoweb. Let's focus on what truly matters.

Everything related to docs.gno.land should be on the standalone website docs.gno.land, except for the .github/workflows trigger, in case a crontab is insufficient.

Reply:
I understand your position, i first update following your comment asking no js, or for json or yaml ( cf: https://github.com/gnolang/gno/pull/1801#discussion_r1580648673 )

But i guess the easiest solutions is to keep a .js file inside the gnolang/docs.gno.land repository and make a PR on the docs.gno.land repository when it's required to update the sidebars.